sbt-dumpschema/src/sbt-test/sbt-dumpschema/simple/project/plugins.sbt

```diff
@@ -0,0 +1,7 @@
+//sys.props.get("plugin.version") match {
```

Comment:
To be removed before the plugin is merged

Reply:
This is a setup for automatic sbt plugin testing, before PR is merged, this should be working